CHANGELOG.md

```diff
@@ -61,6 +61,13 @@ New modules
 Other minor additions
 ---------------------
 
+* Added new proofs to `Data.Nat.DivMod`:
+  ```agda
+  m<n⇒m/n≡0 : ∀ {m n n≢0} → m < n → (m / n) {n≢0} ≡ 0
+  m/n≡1+[m∸n]/n : ∀ {m n n≢0} → m ≥ n → (m / n) {n≢0} ≡ 1 + ((m ∸ n) / n) {n≢0}
+  /-cancelˡ : ∀ m n o {o≢0} {mo≢0} → ((m * n) / (m * o)) {mo≢0} ≡ (n / o) {o≢0}
```

Comment:
In the CHANGELOG for readability we usually miss out the universally quantification of arguments, e.g. the first one would become just `m < n → (m / n) {n≢0} ≡ 0`. Otherwise looks good to go :+1:

Reply:
Done :) Cheers for the comments Matthew!